geopyspark/geotrellis/layer.py

```diff
@@ -232,6 +266,55 @@ def to_png_rdd(self, color_map):
 
         return create_python_rdd(self.pysc, result, ser)
 
+    def to_geotiff_rdd(self,
+                       storage_method=StorageMethod.STRIPED,
+                       rows_per_strip=None,
+                       tile_dimensions=(256, 256),
```

Comment:
The default option scala side is [striped](https://github.com/locationtech/geotrellis/blob/master/raster/src/main/scala/geotrellis/raster/io/geotiff/GeoTiffOptions.scala#L39). I don't know that this is a wrong choice to go with tiled here, curious what your thoughts where.